src/js/languages/default.json

```diff
@@ -161,6 +161,9 @@
   "tabs": {
     "tabContents": "Tab Contents"
   },
+  "tag": {
+    "removeLabel": "Remove {name} {value}"
```

Comment:
This probably is a bit simplistic. I think we probably need 3 variations of this message based on whether name or value is defined or both. And the ':' is there in the visible rendering for a reason. Without it these can get confusing.